charts/.helmignore

```diff
@@ -1,9 +0,0 @@
-# Patterns to ignore when building packages.
-# This supports shell glob matching, relative path matching, and
-# negation (prefixed with !). Only one pattern per line.
-.DS_Store
-# Common VCS dirs
-.git/
-.gitignore
-values?*.yaml
```

Comment:
mmhh why would you remove this? isn't that correct?

Reply:
Actually i have  added it inside charts/daps-server/
so removed from charts/